### PR TITLE
feat: saveRequestFiles support options

### PR DESCRIPTION
**Pull request by okoala**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: test/fixtures/apps/dynamic-option/app/controller/upload.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    async index(ctx) {
+      try {
+        const file = await ctx.saveRequestFiles({
+          limits: {
+            fileSize: 10000
+          }
+        });
+        ctx.body = file;
+      } finally {
+        ctx.cleanupRequestFiles();
```
Comment (fengmk2) [inline test/fixtures/apps/dynamic-option/app/controller/upload.js]: await 加上吧

Comment (okoala) [reply test/fixtures/apps/dynamic-option/app/controller/upload.js]: 这个加不加无所谓，同步和异步的区别～

File: test/file-mode.test.js [hunk 2]
```diff
@@ -356,7 +356,6 @@ describe('test/file-mode.test.js', () => {
       ];
       const shouldKeepDirs = [
         path.join(app.config.multipart.tmpdir, moment().subtract(2, 'years').format('YYYY/MM/DD/HH')),
-        path.join(app.config.multipart.tmpdir, moment().subtract(8, 'days').format('YYYY/MM/DD/HH')),
```
Comment (okoala) [inline test/file-mode.test.js]: 这个用例不正确，干掉了～

**Comment by fengmk2** [comment]
+1

File: test/fixtures/apps/dynamic-option/app/controller/upload.js (hunk 1 above)
Comment (atian25) [reply test/fixtures/apps/dynamic-option/app/controller/upload.js]: 有区别，不 await 的话出问题就是 unrejectpromise 了

**Pull request merged**